minijinja/src/filters.rs

```diff
@@ -1467,6 +1467,58 @@ mod builtins {
         Ok(Value::from(rv))
     }
 
+    /// Chain two or more iterable objects as a single iterable object.
+    ///
+    /// If all the individual objects are dictionaries, then the final chained object
+    /// also acts like a dictionary -- you can lookup a key, or iterate over the keys
+    /// etc. Note that the dictionaries are not merged, so if there are duplicate keys,
+    /// then the lookup will return the value from the last matching dictionary in the
+    /// chain, but the iteration will yield all the duplicate keys.
```

Comment:
Do you have to reword this part as dicts are now actually merged when iterator is invoked?